packages/paste-customization/src/types/CustomTheme.ts

```diff
@@ -2,5 +2,7 @@ import type {GenericThemeShape} from '@twilio-paste/theme';
 import type {PasteCustomCSS} from './PasteCSS';
 
 export type CustomTheme = Partial<GenericThemeShape> & {
-  elements?: PasteCustomCSS;
+  elements?: {
+    [key: string]: PasteCustomCSS;
```

Comment:
is this supposed to be `{[key: string]: PasteCustomCSS} | PasteCustomCSS` or did I read that wrong?

Reply:
Nah, it was me doing a boo boo. Elements should always have a top level key, which is the name of the element.

```
elements={{
  THE_ELEMENT: {...theCSS},
  CARD: {...theCardCSS},
  ALERT: {...theAlertCSS}
}}
```